src/libs/OptionsListUtils.js

```diff
@@ -516,10 +529,8 @@ function createOption(accountIDs, personalDetails, report, reportActions = {}, {
         const lastMessageTextFromReport = getLastMessageTextForReport(report);
         const lastActorDetails = personalDetailMap[report.lastActorAccountID] || null;
         const lastActorDisplayName =
-            hasMultipleParticipants && lastActorDetails && lastActorDetails.accountID !== currentUserAccountID
-                ? lastActorDetails.firstName || PersonalDetailsUtils.getDisplayNameOrDefault(lastActorDetails)
-                : '';
-        let lastMessageText = lastActorDisplayName ? `${lastActorDisplayName}: ${lastMessageTextFromReport}` : lastMessageTextFromReport;
+            hasMultipleParticipants && lastActorDetails && lastActorDetails.accountID !== currentUserAccountID ? lastActorDetails.firstName || lastActorDetails.displayName : '';
+        let lastMessageText = lastMessageTextFromReport;
```

Comment:
The `lastMessageText` with concatenated `lastActorDisplayName` is used later for multiple-user chats:

```js
} else if (result.isChatRoom || result.isPolicyExpenseChat) {
    result.alternateText = showChatPreviewLine && !forcePolicyNamePreview && lastMessageText ? lastMessageText : subtitle;
}
```

Will it work correctly after this change? Is the `lastActorDisplayName` now integrated into `lastMessageTextFromReport` for such reports as well?

Reply:
yep, we don't want to concatenate `lastMessageText` with `lastActorDisplayName` for every message - we have the same logic in SidebarUtils
https://github.com/Expensify/App/blob/0b74586f3af8ccf913e8c6f0467b7c3b4fe635bb/src/libs/SidebarUtils.ts#L401-L403
on your question - I don't see any regression so far. This logic for displaying the previews for reports is still being worked on and some follow-ups and reactors are already planned.

Reply:
Here is the regression I'm talking about:

1. As user A, go to 1:1 chat with user B
2. Send a message
3. Add a reply in thread
4. As user B, add a message to the same thread
5. As user A, open the search and look for this thread

Before this PR changes, the subtext contains User B as the author of the last message:
<img width="1512" alt="image" src="https://github.com/Expensify/App/assets/12595293/a6d82200-0f12-428e-adeb-8e3459edc9c1">

After, there is only message, without User B as an author:
<img width="1512" alt="image" src="https://github.com/Expensify/App/assets/12595293/32bfdd0e-f9ca-4a2c-9a31-12e97aae9aeb">

Reply:
fixed